website/docs/commands/export.md

```diff
@@ -17,6 +17,23 @@ You can pass it explicitly or set it globally by running:
 scala-cli config power true
 :::
 
+:::caution
+The `export` sub-command is an experimental feature.
+
+Please bear in mind that non-ideal user experience should be expected.
+If you encounter any bugs or have feedback to share, make sure to reach out to the maintenance team
+on [GitHub](https://github.com/VirtusLab/scala-cli).
+
+Experimental features' warnings can be suppressed with the `--suppress-experimental-warning` option.
+```bash ignore
+scala-cli --power export --suppress-experimental-warning
+```
+Alternatively, you can configure the warnings to be suppressed globally by running:
+```bash
+scala-cli config suppress-warning.experimental-features true
+```
```

Comment:
Maybe it will be sufficient to mention only just once in the section `commands/basic#logging` that exists this option to disable this warning. And not repeat this information, for each experimental command/option/directive

Reply:
I removed it completely for now, and will add a verbosity guide to the docs in a separate PR.